### ISSUE TITLE
fix wiki page about Ceph on XCP-ng 7.5

### ISSUE THREAD

**Issue by northbear** [xcp-ng/xcp#47]
For wiki page: https://github.com/xcp-ng/xcp/wiki/Ceph-on-XCP-ng-7.5-or-later

To install qemu-dp also required 'base' repo of CentOS. 
` yum install qemu-dp --enablerepo="xcp-ng-extras_testing*,base"`
It needs to install dependencies: librados2 and librbd1.

**Comment by northbear** [comment]
And actually It isn't clear what to do after installing qemu-dp. 'xe sr-introduce' and 'xe sr-create' still have no type of device 'rbd' listed or something like that. So actual questions are:
* Should ceph client be installed? If yes, What is recommended way to install ceph client on XCP-ng 7.5?
* Should be plugin RBDSR (https://github.com/rposudnevskiy/RBDSR) be installed? (I suppose it should be https://github.com/rposudnevskiy/RBDSR/tree/v3.0. But I'm not sure)

**Comment by olivierlambert** [comment]
IIRC you shouldn't have to enable CentOS repo in any case. But @stormi is the distro master, he'll answer when he's back :)

**Comment by northbear** [comment]
@oallart It looks like actually you've still used the plugin v2.0. For the plugin v3.0 update of qemu-dp is mandatory.

**Comment by stormi** [comment]
I'm all for improving the wiki page, but I never used Ceph and I don't have the hardware to test. I asked several times on the forum for help but no one help for now. I worked with @rposudnevskiy to patch qemu-dp, but I never really got proper explanations about why it was needed and how it took place in the big picture.

From what I understand, you need https://github.com/rposudnevskiy/RBDSR/tree/v3.0 indeed.

The librados2 and librbd1 packages from CentOS are quite old, I've been told you'd better use newer versions from dedicated repositories made for Ceph on CentOS.

Help very welcome to improve that wiki page and make it a good step by step tutorial!

**Comment by northbear** [comment]
To be honest I'd not succeed to integrate ceph storage by RBDSR plugin v3.0, so actually I cannot help much. I don't know proper way to reach working configuration. 
About points/questions I've denoted in second post I can now say definitely: 
1. Proper ceph client will be installed by script that installing RBDSR plugin. At least, It works so now... 
2. At the end of all, it should be used RBDSR plugin v3.0. But as I understand, at this stage It doesn't work properly... 

If I've succeed with getting of working configuration ceph storage+RBDSRv3.0+XCP-ng7.5, I'm ready to  provide guide on it.
